client/state/data-layer/wpcom/activity-log/update-credentials/index.js

```diff
@@ -144,33 +122,25 @@ export const failure = ( action, error ) => ( dispatch, getState ) => {
 			port: action.credentials.port,
 			protocol: action.credentials.protocol,
 			user: action.credentials.user,
-		} );
-
-		dispatch(
-			hasActiveHappychatSession( getState() )
-				? withAnalytics( tracksEvent, sendEvent( message ) )
-				: tracksEvent
-		);
-	};
+		} )
+	);
 
 	switch ( error.code ) {
 		case 'service_unavailable':
 			announce(
 				i18n.translate(
-					'A error occurred when we were trying to validate your site information. Please make sure your credentials and host URL are correct and try again. If you need help, please click on the support chat link.'
+					'A error occurred when we were trying to validate your site information. ' +
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.